src/cap-description.adoc

```diff
@@ -118,6 +118,11 @@ memory if the authorising capability also grants <<w_perm>>.
 [#x_perm,reftext="X-permission"]
 Execute Permission (X):: Allow instruction execution.
 
+[#lm_perm,reftext="LM-permission"]
+Load Mutable Permission (LM):: Allow loads of capabilities with <<w_perm>> via a read-only authorizing capability one.
+Clearing this permission allows sharing a read-only version of e.g. a tree structure without making a copy.
+If this permission is not set, capability loads authorized by this capability will have resulting permissions as if an implicit <<ACPERM>> clearing <<w_perm>> and <<lm_perm>> was executed.
```

Comment:
I think it's a useful way of describing it, it's just the same as describing the CSR updates as using SCADDR, so that the full semantics are described - however - it's not accurate because ACPERM clears the tag if the cap is selaed but in this case the cap is _not modified_ if sealed. Also ACPERM still operates if the tag=0, and this doesn't do anything.

I've suggested a change to the text above accordingly.